useful-scripts/misc-test-utils/src/services/releaseService.ts

```diff
@@ -72,5 +76,44 @@ const releaseService = {
       publishMethod: 'Immediate',
     };
   },
+
+  addContentSection: async (releaseId: string): Promise<string> => {
+    const res = await adminApi.post(
+      `/api/release/${releaseId}/content/sections/add`,
+      {
+        order: 0,
+      },
+    );
+    const sectionId = res.data.id;
+    return sectionId;
+  },
+  addTextBlock: async (
+    releaseId: string,
+    sectionId: string,
+  ): Promise<string> => {
+    const res = await adminApi.post(
+      `/api/release/${releaseId}/content/section/${sectionId}/blocks/add`,
+      {
+        body: '',
+        order: 0,
+        type: 'HtmlBlock',
+      },
+    );
+    const blockId = res.data.id;
+    return blockId;
+  },
+  addTextContent: async (
+    releaseId: string,
+    sectionId: string,
+    blockId: string,
+  ): Promise<boolean> => {
+    await adminApi.put(
+      `/api/release/${releaseId}/content/section/${sectionId}/block/${blockId}`,
+      {
+        body: `<p>${faker.lorem.lines(100)}</p>`,
+      },
+    );
+    return true;
```

Comment:
If you don't want to have to return something arbitrary like `true`, I think you can just get this function to return `Promise` rather than `Promise<Something>`

Reply:
I think `Promise` requires at least 1 type argument unfortunately `Generic type 'Promise<T>' requires 1 type argument(s).ts(2314)`

Reply:
Maybe try a Promise<void> in the future?

Reply:
Done. Looks a lot cleaner now